x-pack/plugins/upgrade_assistant/public/application/components/shared/search_bar/group_by_filter.tsx

```diff
@@ -14,10 +14,10 @@ import { GroupByOption } from '../../types';
 
 const LocalizedOptions: { [option: string]: string } = {
   message: i18n.translate('xpack.upgradeAssistant.checkupTab.controls.groupByBar.byIssueLabel', {
-    defaultMessage: 'by issue',
+    defaultMessage: 'By issue',
```

Comment:
@debadair since you suggested capitalizing `Critical` and `Warning`, it seemed like we should also capitalize the `By issue` and `By index` filters on the ES deprecations page. Let me know if you disagree.